gto-support-util/src/main/java/org/ccci/gto/android/common/util/content/Context.kt

```diff
@@ -4,16 +4,20 @@ import android.content.Context
 import android.content.res.Configuration
 import android.os.Build
 import android.os.LocaleList
-import org.ccci.gto.android.common.util.os.toTypedArray
+import org.ccci.gto.android.common.util.os.locales
 import java.util.Locale
 
+@OptIn(ExperimentalStdlibApi::class)
 fun Context.localize(vararg locales: Locale, includeExisting: Boolean = true): Context = when {
     locales.isEmpty() -> this
     Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1 -> createConfigurationContext(Configuration().apply {
         when {
             Build.VERSION.SDK_INT >= Build.VERSION_CODES.N -> setLocales(
                 LocaleList(
-                    *locales, *(if (includeExisting) resources.configuration.locales.toTypedArray() else emptyArray())
+                    *buildList<Locale> {
```

Comment:
They are in the process of reworking the type inference detection in Kotlin, the `<Locale>` generic here is reported as not being necessary in Android Studio, but removing it causes a compile error from being unable to infer the type.

The reason for this is that they have enabled the new type inference logic on the editor, but are still using the old inference logic for the compiler while they work out any bugs in the new logic.

This is something else that will be a refactoring once Kotlin 1.4 is released